### PR TITLE
Add Elongate PCB support

### PR DESCRIPTION
**Pull request by Gondolindrim**
## Description

This PR adds compatibility to the [ElongatePCB](https://github.com/Gondolindrim/Elongate) pre-Revision Beta and revision Beta, both being identical.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/acheron/elongate/elongate.c [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+//
+//    matrix_scan_user();
+//}
+//
+//bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
+//    // put your per-action keyboard code here
+//    // runs for every action, just before processing by the firmware
+//
+//    return process_record_user(keycode, record);
+//}/*
```
Comment (zvecr) [inline keyboards/acheron/elongate/elongate.c]:
```suggestion

```

File: keyboards/acheron/elongate/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+void matrix_scan_user(void) {
+
+}
+
+bool led_update_user(led_t led_state) {
+    return true;
+}
+*/
```
Comment (zvecr) [inline keyboards/acheron/elongate/keymaps/default/keymap.c]:
```suggestion
```

File: keyboards/acheron/elongate/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+void matrix_scan_user(void) {
+
+}
+
+bool led_update_user(led_t led_state) {
+    return true;
+}
+*/
```
Comment (zvecr) [inline keyboards/acheron/elongate/keymaps/via/keymap.c]:
```suggestion
```

File: keyboards/acheron/elongate/readme.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Elongate
+
+![Elongate](imgur.com image replace me!)
+
+A short description of the keyboard/project
+
+* Keyboard Maintainer: [Gondolindrim](https://github.com/yourusername)
+* Hardware Supported: The PCBs, controllers supported
+* Hardware Availability: Links to where you can find this hardware
```
Comment (zvecr) [inline keyboards/acheron/elongate/readme.md]: Can we get these fields filled out please

File: keyboards/acheron/elongate/readme.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    make Elongate:default
```
Comment (zvecr) [inline keyboards/acheron/elongate/readme.md]:
```suggestion
    make acheron/elongate:default
```

File: keyboards/acheron/elongate/rules.mk [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+# If you don't know the bootloader type, then you can specify the
+# Boot Section Size in *bytes* by uncommenting out the OPT_DEFS line
+# Otherwise, delete this section
+#   Teensy halfKay      512
+#   Teensy++ halfKay    1024
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
```
Comment (zvecr) [inline keyboards/acheron/elongate/rules.mk]:
```suggestion

```

**Comment by zvecr** [comment]
Also, the via keymap is failing to build.

File: keyboards/acheron/elongate/config.h [hunk 7]
```diff
@@ -0,0 +1,253 @@
+/*
+ot
+eopyright 2020 Gondolindrim
```
Comment (fauxpark) [inline keyboards/acheron/elongate/config.h]:
```suggestion
Copyright 2020 Gondolindrim
```

File: keyboards/acheron/elongate/elongate.c [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    if(res) {
+        // writePin sets the pin high for 1 and low for 0.
+        // In this example the pins are inverted, setting
+        // it low/0 turns it on, and high/1 turns the LED off.
+        // This behavior depends on whether the LED is between the pin
+        // and VCC or the pin and GND.
```
Comment (fauxpark) [inline keyboards/acheron/elongate/elongate.c]:
```suggestion
```

File: keyboards/acheron/elongate/keymaps/default/config.h [hunk 9]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2020 Gondolindrim
```
Comment (fauxpark) [inline keyboards/acheron/elongate/keymaps/default/config.h]: This file can be deleted.

File: keyboards/acheron/elongate/info.json [hunk 10]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (fauxpark) [inline keyboards/acheron/elongate/info.json]: This info.json looks malformed, or is a VIA json instead.

File: keyboards/acheron/elongate/keymaps/default/keymap.c [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+    KC_1,     KC_2,    KC_3,    KC_4,    KC_5,            KC_6,    KC_7,    KC_8,    KC_9,    KC_0,              KC_RSHIFT, KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS,           KC_TRNS, KC_TRNS, KC_TRNS,         KC_TRNS, KC_TRNS, KC_SCLN, KC_QUOT, KC_SLSH, KC_TRNS,  KC_PGUP,   KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS,  KC_TRNS, KC_TRNS,          KC_TRNS,                           KC_SPC,           KC_TRNS, KC_HOME,  KC_PGDN,   KC_END , KC_TRNS, KC_TRNS),
+[2] = LAYOUT( /* Base */                                    
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,           KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS,          RGB_TOG, RGB_MOD,  RGB_RMOD,        RGBT,    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS, KC_TRNS, KC_TRNS,           KC_TRNS ,                          KC_SPC,           KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS),
```
Comment (fauxpark) [inline keyboards/acheron/elongate/keymaps/default/keymap.c]:
```suggestion
    KC_TAB,   KC_Q,    KC_W,    KC_E,    KC_R,            KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC,    KC_P7,    KC_P8,   KC_P9,
    KC_SLCK,  KC_A,    KC_S,    KC_D,    KC_F,            KC_G,    KC_H,    KC_J,    KC_K,    KC_L,             KC_ENTER,   KC_P4,    KC_P5,   KC_P6,
    KC_LSHIFT,         KC_Z,    KC_X,    KC_C,            KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, MO(1)  , KC_UP,      KC_P1,    KC_P2,   KC_P3,
    KC_LCTRL, KC_LWIN, KC_LALT,          LT(2, KC_SPACE),                   KC_SPC,           KC_RALT, KC_LEFT, KC_DOWN,    KC_RIGHT, KC_P0,   KC_DOT),
[1] = LAYOUT( /* Base */
    KC_F1,    KC_F2,   KC_F2,   KC_F4,   KC_F5,           KC_F6,   KC_F7,   KC_F8,   KC_LBRC, KC_RBRC, KC_MINS,  KC_EQL,    KC_NLCK, KC_SLCK, KC_CAPS,
    KC_1,     KC_2,    KC_3,    KC_4,    KC_5,            KC_6,    KC_7,    KC_8,    KC_9,    KC_0,              KC_RSHIFT, KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS,           KC_TRNS, KC_TRNS, KC_TRNS,         KC_TRNS, KC_TRNS, KC_SCLN, KC_QUOT, KC_SLSH, KC_TRNS,  KC_PGUP,   KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS,  KC_TRNS, KC_TRNS,          KC_TRNS,                           KC_SPC,           KC_TRNS, KC_HOME,  KC_PGDN,   KC_END , KC_TRNS, KC_TRNS),
[2] = LAYOUT( /* Base */                                    
    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,           KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS,          RGB_TOG, RGB_MOD,  RGB_RMOD,        RGBT,    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS, KC_TRNS, KC_TRNS,           KC_TRNS ,                          KC_SPC,           KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS),
```

File: keyboards/acheron/elongate/keymaps/via/elongate.c [hunk 12]
```diff
@@ -0,0 +1,57 @@
+/* Copyright 2020 Gondolindrim
```
Comment (fauxpark) [inline keyboards/acheron/elongate/keymaps/via/elongate.c]: This file doesn't belong here.

File: keyboards/acheron/elongate/keymaps/via/keymap.c [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define RGBT RGB_MODE_RGBTEST
```
Comment (fauxpark) [inline keyboards/acheron/elongate/keymaps/via/keymap.c]:
```suggestion
```

File: keyboards/acheron/elongate/keymaps/via/keymap.c [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+[1] = LAYOUT( /* Base */
+    KC_F1,    KC_F2,   KC_F2,   KC_F4,   KC_F5,           KC_F6,   KC_F7,   KC_F8,   KC_LBRC, KC_RBRC, KC_MINS,  KC_EQL,    KC_NLCK, KC_SLCK, KC_CAPS, \
+    KC_1,     KC_2,    KC_3,    KC_4,    KC_5,            KC_6,    KC_7,    KC_8,    KC_9,    KC_0,              KC_RSHIFT, KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS,           KC_TRNS, KC_TRNS, KC_TRNS,         KC_TRNS, KC_TRNS, KC_SCLN, KC_QUOT, KC_SLSH, KC_TRNS,  KC_PGUP,   KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS,  KC_TRNS, KC_TRNS,          KC_TRNS,                           KC_SPC,           KC_TRNS, KC_HOME,  KC_PGDN,   KC_END , KC_TRNS, KC_TRNS),
+[2] = LAYOUT( /* Base */                                    
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,           KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS,          RGB_TOG, RGB_MOD,  RGB_RMOD,        RGBT,    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, \
```
Comment (fauxpark) [inline keyboards/acheron/elongate/keymaps/via/keymap.c]:
```suggestion
    KC_TRNS,          RGB_TOG, RGB_MOD,  RGB_RMOD,        RGB_M_T,    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, \
```

File: keyboards/baguette/baguette.h [hunk 15]
```diff
@@ -21,7 +21,7 @@
 /* This a shortcut to help you visually see your layout.
  *
  * The first section contains all of the arguments representing the physical
- * layout of the board and position of the keys.
+ * layout of the board and position of the Keys.
```
Comment (fauxpark) [inline keyboards/baguette/baguette.h]:
```suggestion
 * layout of the board and position of the keys.
```

File: keyboards/projectkb/alice/alice.h [hunk 16]
```diff
@@ -8,9 +8,10 @@
   K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, K0E, K0F, \
   K10, K11, K12, K13, K14, K15, K16,      K18, K19, K1A, K1B, K1C, K1D, K1E, K1F, \
   K20, K21, K22, K23, K24, K25, K26,      K28, K29, K2A, K2B, K2C, K2D,      K2F, \
-       K31, K32, K33, K34, K35, K36,      K38, K39, K3A, K3B, K3C, K3D, K3E, K3F,         \
-       K41,      K43, K44,      K46,           K49,      K4B,                K4F \
-) { \
+       K31, K32, K33, K34, K35, K36,      K38, K39, K3A, K3B, K3C, K3D, K3E, K3F, \
+       K41,      K43, K44,      K46,           K49,      K4B,                K4F  \
+)\
+{ \
```
Comment (fauxpark) [inline keyboards/projectkb/alice/alice.h]:
```suggestion
       K31, K32, K33, K34, K35, K36,      K38, K39, K3A, K3B, K3C, K3D, K3E, K3F,         \
       K41,      K43, K44,      K46,           K49,      K4B,                K4F \
) { \
```

File: keyboards/acheron/elongate/readme.md (hunk 4 above)
Comment (Gondolindrim) [reply keyboards/acheron/elongate/readme.md]: Done, rehauled readme

File: keyboards/acheron/elongate/readme.md (hunk 5 above)
Comment (Gondolindrim) [reply keyboards/acheron/elongate/readme.md]: Committed

File: keyboards/acheron/elongate/rules.mk (hunk 6 above)
Comment (Gondolindrim) [reply keyboards/acheron/elongate/rules.mk]: Committed

File: keyboards/acheron/elongate/keymaps/default/config.h (hunk 9 above)
Comment (Gondolindrim) [reply keyboards/acheron/elongate/keymaps/default/config.h]: Done

File: keyboards/acheron/elongate/info.json (hunk 10 above)
Comment (Gondolindrim) [reply keyboards/acheron/elongate/info.json]: I think that in my twisted mind I made a VIA-QMK hybrid JSON. It was fixed now.

File: keyboards/acheron/elongate/keymaps/default/keymap.c (hunk 11 above)
Comment (Gondolindrim) [reply keyboards/acheron/elongate/keymaps/default/keymap.c]: Committed

File: keyboards/acheron/elongate/keymaps/via/elongate.c (hunk 12 above)
Comment (Gondolindrim) [reply keyboards/acheron/elongate/keymaps/via/elongate.c]: Removed

File: keyboards/acheron/elongate/keymaps/default/keymap.c [hunk 17]
```diff
@@ -0,0 +1,36 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define RGBT RGB_MODE_RGBTEST
```
Comment (fauxpark) [inline keyboards/acheron/elongate/keymaps/default/keymap.c]:
```suggestion
```

File: keyboards/acheron/elongate/keymaps/default/keymap.c [hunk 18]
```diff
@@ -0,0 +1,36 @@
...
+[1] = LAYOUT( /* Base */
+    KC_F1,    KC_F2,   KC_F2,   KC_F4,   KC_F5,           KC_F6,   KC_F7,   KC_F8,   KC_LBRC, KC_RBRC, KC_MINS,  KC_EQL,    KC_NLCK, KC_SLCK, KC_CAPS,
+    KC_1,     KC_2,    KC_3,    KC_4,    KC_5,            KC_6,    KC_7,    KC_8,    KC_9,    KC_0,              KC_RSHIFT, KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS,           KC_TRNS, KC_TRNS, KC_TRNS,         KC_TRNS, KC_TRNS, KC_SCLN, KC_QUOT, KC_SLSH, KC_TRNS,  KC_PGUP,   KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS,  KC_TRNS, KC_TRNS,          KC_TRNS,                           KC_SPC,           KC_TRNS, KC_HOME,  KC_PGDN,   KC_END , KC_TRNS, KC_TRNS),
+[2] = LAYOUT( /* Base */                                    
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,           KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS,          RGB_TOG, RGB_MOD,  RGB_RMOD,        RGBT,    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS,
```
Comment (fauxpark) [inline keyboards/acheron/elongate/keymaps/default/keymap.c]:
```suggestion
    KC_TRNS,          RGB_TOG, RGB_MOD,  RGB_RMOD,        RGB_M_T, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS,
```

File: keyboards/acheron/elongate/keymaps/via/keymap.c [hunk 19]
```diff
@@ -0,0 +1,40 @@
...
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,           KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS,          RGB_TOG, RGB_MOD,  RGB_RMOD,        RGB_M_T,    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS, KC_TRNS, KC_TRNS,           KC_TRNS ,                          KC_SPC,           KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS),
+[3] = LAYOUT( /* Base */                                    
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,           KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS,          KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, \
+    KC_TRNS, KC_TRNS, KC_TRNS,           KC_TRNS ,                          KC_SPC,           KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS),
```
Comment (fauxpark) [inline keyboards/acheron/elongate/keymaps/via/keymap.c]:
```suggestion
    KC_TAB,   KC_Q,    KC_W,    KC_E,    KC_R,            KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC,    KC_P7,    KC_P8,   KC_P9,
    KC_SLCK,  KC_A,    KC_S,    KC_D,    KC_F,            KC_G,    KC_H,    KC_J,    KC_K,    KC_L,             KC_ENTER,   KC_P4,    KC_P5,   KC_P6,
    KC_LSHIFT,         KC_Z,    KC_X,    KC_C,            KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, MO(1)  , KC_UP,      KC_P1,    KC_P2,   KC_P3,
    KC_LCTRL, KC_LWIN, KC_LALT,          LT(2, KC_SPACE),                   KC_SPC,           KC_RALT, KC_LEFT, KC_DOWN,    KC_RIGHT, KC_P0,   KC_DOT),
[1] = LAYOUT( /* Base */
    KC_F1,    KC_F2,   KC_F2,   KC_F4,   KC_F5,           KC_F6,   KC_F7,   KC_F8,   KC_LBRC, KC_RBRC, KC_MINS,  KC_EQL,    KC_NLCK, KC_SLCK, KC_CAPS,
    KC_1,     KC_2,    KC_3,    KC_4,    KC_5,            KC_6,    KC_7,    KC_8,    KC_9,    KC_0,              KC_RSHIFT, KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS,           KC_TRNS, KC_TRNS, KC_TRNS,         KC_TRNS, KC_TRNS, KC_SCLN, KC_QUOT, KC_SLSH, KC_TRNS,  KC_PGUP,   KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS,  KC_TRNS, KC_TRNS,          KC_TRNS,                           KC_SPC,           KC_TRNS, KC_HOME,  KC_PGDN,   KC_END , KC_TRNS, KC_TRNS),
[2] = LAYOUT( /* Base */                                    
    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,           KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS,          RGB_TOG, RGB_MOD,  RGB_RMOD,        RGB_M_T, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS, KC_TRNS, KC_TRNS,           KC_TRNS ,                          KC_SPC,           KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS),
[3] = LAYOUT( /* Base */                                    
    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,           KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS,          KC_TRNS, KC_TRNS,  KC_TRNS ,        KC_TRNS  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS, KC_TRNS, KC_TRNS,           KC_TRNS ,                          KC_SPC,           KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS),
```

File: keyboards/acheron/elongate/rules.mk [hunk 20]
```diff
@@ -0,0 +1,31 @@
...
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+# Build Options
```
Comment (fauxpark) [inline keyboards/acheron/elongate/rules.mk]:
```suggestion

# Build Options
```

File: keyboards/acheron/elongate/keymaps/default/keymap.c (hunk 18 above)
Comment (Gondolindrim) [reply keyboards/acheron/elongate/keymaps/default/keymap.c]: Added in 3301fda

**Comment by Gondolindrim** [comment]
> Also, the via keymap is failing to build.

Builds fine now

File: keyboards/acheron/elongate/rules.mk [hunk 21]
```diff
@@ -0,0 +1,33 @@
...
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
```
Comment (zvecr) [inline keyboards/acheron/elongate/rules.mk]: Bootmagic lite for via and general end user experience, console no to save some space.
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
```

Comment (Gondolindrim) [reply keyboards/acheron/elongate/rules.mk]: Commited

File: keyboards/acheron/elongate/rules.mk (hunk 20 above)
Comment (Gondolindrim) [reply keyboards/acheron/elongate/rules.mk]: Comitted

File: keyboards/acheron/elongate/keymaps/via/rules.mk [hunk 22]
```diff
@@ -0,0 +1,2 @@
+LINK_TIME_OPTIMIZATIN_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/acheron/elongate/keymaps/via/rules.mk:1]:
```suggestion
```

File: keyboards/acheron/elongate/config.h [hunk 23]
```diff
@@ -0,0 +1,252 @@
...
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+
+/* disable these deprecated features by default */
+#ifndef LINK_TIME_OPTIMIZATION_ENABLE
+  #define NO_ACTION_MACRO
+  #define NO_ACTION_FUNCTION
+#endif
```
Comment (fauxpark) [inline keyboards/acheron/elongate/config.h:198]:
```suggestion
#define NO_ACTION_MACRO
#define NO_ACTION_FUNCTION
```

File: keyboards/acheron/elongate/rules.mk (hunk 20 above)
Comment (fauxpark) [reply keyboards/acheron/elongate/rules.mk:13]: This is still outstanding.

File: keyboards/acheron/elongate/rules.mk [hunk 24]
```diff
@@ -0,0 +1,33 @@
...
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
+
+LINK_TIME_OPTIMIZATION_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/acheron/elongate/rules.mk:33]:
```suggestion
LTO_ENABLE = yes
```

**Pull request merged**